### PR TITLE
fix: manifest file for firefox compatibility 

### PR DESCRIPTION
**Pull request by kilted-andres**
## fixes KILTProtocol/ticket#3619

Firefox does not support `service_workers` yet, which are the chrome's default background scripts.

Here is some documentation on it:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/background

### CODE REVIEW

File: src/static/manifest.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "default_popup": "popup.html"
   },
   "background": {
-    "service_worker": "js/backgroundScript.js"
+    "service_worker": "js/backgroundScript.js",
+    "scripts": ["js/backgroundScript.js"]
```
Comment (Dudleyneedham) [inline src/static/manifest.json:19]: Does this work with chrome?

Comment (Dudleyneedham) [reply src/static/manifest.json:19]: Will chrome also make a stink if this is there?

**Review by Dudleyneedham — COMMENTED** [review]
Minor comment

File: src/static/manifest.json (hunk 1 above)
Comment (kilted-andres) [reply src/static/manifest.json:19]: It works on chrome as well.

**Review by Dudleyneedham — APPROVED** [review]
LGTM

**Pull request merged**